docs/src/CommutativeAlgebra/module_localizations.md

```diff
@@ -0,0 +1,81 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["module_localizations.md"]
+```
+
+# Localizations of modules over computable rings
+
+For localizations of modules, there exists a generic implementation of 
+the common methods such as membership tests, kernel computations, etc. 
+based on the work of Barakat, Posur, et. al; see [Pos18](@cite).
+
+Let $R$ be a ring of type `<:Ring`, $U \subset R$ a multiplicative set of type `<:AbsMultSet` 
+and $S = R[U^{-1}]$ the localization of $R$ at $U$. Recall that $R$ 
+is *computable* if one can compute *syzygies* and *lifts* over $R$. 
+The results from [Pos18](@cite), Theorem 3.9, assert that then also the localization $S$ is 
+computable, provided that there exists a solution to the *localization problem* 
+(Definition 3.8, [Pos18](@cite) and below). 
+
+The user who wishes to use the generic code for 
+localizations therefore has to make sure the following two 
+requirements are met: 
+
+ 1) The code for finitely generated modules and ideals must be functional over ``R``, including the computation of `coordinates` and `kernel`. 
+
+ 2) The user has to solve the *localization problem* by implementing `has_nonepmty_intersection(U::MultSetType, I::IdealType)` for the type `MultSetType` of multiplicative sets and the type `IdealType` of ideals in `R` that they would like to consider.
```

Comment:
This should be a markdown list.

Also, I would not spell out the arguments in the second point. You could however think about using `@ref` to turn it into a link? That would also catch typos ("epmty" -> "empty")

```suggestion
 1. The code for finitely generated modules and ideals must be functional over ``R``, including the computation of `coordinates` and `kernel`. 

 2. The user has to solve the *localization problem* by implementing `has_nonempty_intersection` for the argument types that they would like to consider.
```